lib/oslc_lisp.pl

```diff
@@ -21,13 +21,17 @@
 :- multifile lisp:funct/3.
 
 lisp:funct(send, [IRI, URI], true) :- !,
+  debug(lisp(oslc), 'Sending a POST to  ~w', [URI]),
   oslc_client:post_resource(IRI, URI, []).
 
 lisp:funct(send, [IRI, URI, Options], true) :- !,
+  debug(lisp(oslc), 'Sending a POST to ~w', [URI]),
```

Comment:
Shall we do this for `post_resource`:
`debug(lisp(oslc), 'POSTing [~w] to [~w]', [IRI, URI]),`